invokeai/frontend/cli/arg_parser.py

```diff
@@ -0,0 +1,47 @@
+from argparse import ArgumentParser, Namespace, RawTextHelpFormatter
+from typing import Optional
+
+from invokeai.version import __version__
+
+_root_help = r"""Sets a root directory for the app.
+If omitted, the app will search for the root directory in the following order:
+- The `$INVOKEAI_ROOT` environment variable
+- The currently active virtual environment's parent directory
+- `$HOME/invokeai`"""
+
+_ignore_missing_core_models_help = r"""If set, the app will ignore missing core diffusers conversion models.
+These are required to use checkpoint/safetensors models.
+If you only use diffusers models, you can safely enable this."""
+
+_parser = ArgumentParser(description="Invoke Studio", formatter_class=RawTextHelpFormatter)
+_parser.add_argument("--root", type=str, help=_root_help)
+_parser.add_argument("--ignore_missing_core_models", action="store_true", help=_ignore_missing_core_models_help)
```

Comment:
Doesn't seem like this is getting set in the config

Reply:
Adding an arg to pass a different source_path file to merge_from_file would probably get ahead of most of the issues faced by getting rid of the old args. Not sure how many people used them.

Reply:
Whoops, fixed it not getting set

Reply:
> Adding an arg to pass a different source_path file to merge_from_file would probably get ahead of most of the issues faced by getting rid of the old args. Not sure how many people used them.

Yeah, was thinking about this, but I'm not sure how to make all the pieces work together. Can explore this in a followup.